contracts/vaults/Claim.sol

```diff
@@ -51,15 +51,22 @@ contract Claim is Base {
         emit SubmitClaim(_pid, msg.sender, _beneficiary, _bountyPercentage, _descriptionHash);
     }
 
+    function isChallenged(uint256 _pid) onlyArbitrator external {
+        // TODO: challenge logic in HATVaults.
```

Comment:
todo: so we will have claimIds, and when that is ready so the arg will be "_claimID" here